app/lib/dartdoc/backend.dart

```diff
@@ -296,7 +296,9 @@ class DartdocBackend {
         if (index >= 0) {
           final entry = completedList[index];
           completedList.removeAt(index);
-          versions.remove(Version.parse(entry.runtimeVersion));
+          if (entry.runtimeVersion != null) {
```

Comment:
Is this a really old entity? If so, how about we write a tiny tool to delete it? and remove associated resources...

Maybe a tool to list number of things associated with this, and then delete it? Isn't that better than making exceptions for every case of old data we have in datastore that we're never going to use again?

I assume we're never going to use this again, is that wrong?

Reply:
I think a tool would only duplicate this logic (e.g. we want to preserve the latest serving version, even if it does not have a `runtimeVersion`). While it would be nice to have it as a separate tool, I think the overhead to write and test it does not really provide that much value.